config/environments/development.rb

```diff
@@ -41,12 +52,19 @@
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
+  # Raise exceptions for disallowed deprecations.
+  config.active_support.disallowed_deprecation = :raise
+
+  # Tell Active Support which deprecation messages to disallow.
+  config.active_support.disallowed_deprecation_warnings = []
```

Comment:
Related to setting above, allows us to specify deprecations to not allow - https://edgeguides.rubyonrails.org/configuring.html#config-active-support-disallowed-deprecation-warnings